services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/FlowValidationNegativeSpec.groovy

```diff
@@ -38,11 +38,19 @@ class FlowValidationNegativeSpec extends BaseSpecification {
     Database database
 
     @Unroll
-    def "Flow and switch validation should fail in case of missing rules"() {
+    def "Flow and switch validation should fail in case of missing rules with #flowConfig configuration"() {
         given: "Two flows with #flowConfig configuration"
         def (src, dest) = switches
-        def flowToBreak = northbound.addFlow(flowHelper.randomFlow(src, dest))
-        def intactFlow = northbound.addFlow(flowHelper.randomFlow(src, dest))
+        def flowToBreak = flowHelper.randomFlow(src, dest)
```

Comment:
more compact, more effective and easier to understand (no reassignment happen)  
`def flowToBreak = src == dest ? flowHelper.singleSwitchFlow(src) : flowHelper.randomFlow(src, dest)`

Reply:
Fixed, thanks!